tldraw/tldraw
# Make sure loading screens use dark mode user preference.

We load the user preferences a bit earlier, so that we can make sure that the `LoadingScreen` and `ErrorScreen` also use the correct color and background color based on the dark mode setting.

There's still a brief flash of white screen, but that's before any of our components load, not sure if we can avoid that one.

Solves https://github.com/tldraw/tldraw/issues/1248

### Change Type


### Test Plan

1. Probably best if you throttle your network speed.
2. Reload the page.
3. The asset loading screen should use take your dark mode setting into account.
4. Change the dark mode and try again.

### Release Notes

- Make sure our loading and error screens take dark mode setting into account.
